.github/workflows/e2e-webpack-workflow.yml

```diff
@@ -0,0 +1,65 @@
+on:
+  schedule:
+  - cron: '0 */4 * * *'
+  push:
+    branches:
+    - master
+  pull_request:
+    paths:
+    - .github/workflows/e2e-webpack-workflow.yml
+    - scripts/e2e-setup-ci.sh
+
+name: 'E2E Webpack'
+jobs:
+  chore:
+    name: 'Validating Webpack'
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@master
+
+    - name: 'Use Node.js 10.x'
+      uses: actions/setup-node@master
+      with:
+        version: 10.x
+
+    - name: 'Build the standard bundle'
+      run: |
+        node ./scripts/run-yarn.js build:cli
+
+    - name: 'Running the integration test'
+      run: |
+        source scripts/e2e-setup-ci.sh
+
+        yarn init -p
+        yarn add -D webpack@^5.0.0-alpha.24 webpack-cli@^3.3.8 lodash
```

Comment:
Might be interesting to test that loaders can be properly imported as well (cf https://github.com/yarnpkg/berry/pull/442/files#diff-7328490c8360c3ad0988c3bf427a59a9R40)

Reply:
Added 👍